config/benchmark_configs/climaatmos.yml

```diff
@@ -0,0 +1,21 @@
+FLOAT_TYPE: "Float32"
+approximate_linear_solve_iters: 2
+dt: 120secs
+dt_cloud_fraction: 1hours
+dt_rad: 1hours
+dt_save_state_to_disk: "Inf"
+dt_save_to_sol: "Inf"
+dz_bottom: 30.0
+dz_top: 3000.0
+h_elem: 30
+moist: equil
+output_default_diagnostics: false
```

Comment:
Maybe in a next PR we can switch the diagnostics on for our target jobs?  We need to include them if the table has to be representative of realistic runs.

Reply:
Our goal for the table was not to include diagnostics, since that isn’t included in the performance goals we have. If we’re interested in understanding how long “real runs” will take, I can also add a run with diagnostics

Reply:
The performance goals should include IO: at the end, we want to look at the output.